cryptography/fuzz/fuzz_targets/bls12381_aggregate_operations.rs

```diff
@@ -0,0 +1,285 @@
+    },
+    SignaturesMinSig {
+        signatures: Vec<G1>,
+    },
+    AggregateVerifyMultiplePublicKeysMinPk {
+        public_keys: Vec<G1>,
+        namespace: Option<Vec<u8>>,
+        message: Vec<u8>,
+        signature: G2,
+    },
+    AggregateVerifyMultiplePublicKeysMinSig {
+        public_keys: Vec<G2>,
+        namespace: Option<Vec<u8>>,
+        message: Vec<u8>,
+        signature: G1,
+    },
+    AggregateVerifyMultipleMessagesMinPk {
+        public_key: G1,
+        messages: Vec<(Option<Vec<u8>>, Vec<u8>)>,
+        signature: G2,
+        concurrency: usize,
+    },
+    AggregateVerifyMultipleMessagesMinSig {
+        public_key: G2,
+        messages: Vec<(Option<Vec<u8>>, Vec<u8>)>,
+        signature: G1,
+        concurrency: usize,
+    },
+}
+
+impl<'a> Arbitrary<'a> for FuzzOperation {
+    fn arbitrary(u: &mut Unstructured<'a>) -> Result<Self, arbitrary::Error> {
+        let choice = u.int_in_range(0..=7)?;
+
+        match choice {
+            0 => Ok(FuzzOperation::PublicKeysMinPk {
+                public_keys: arbitrary_vec_g1(u, 0, 20)?,
+            }),
+            1 => Ok(FuzzOperation::PublicKeysMinSig {
+                public_keys: arbitrary_vec_g2(u, 0, 20)?,
+            }),
+            2 => Ok(FuzzOperation::SignaturesMinPk {
+                signatures: arbitrary_vec_g2(u, 0, 20)?,
+            }),
+            3 => Ok(FuzzOperation::SignaturesMinSig {
+                signatures: arbitrary_vec_g1(u, 0, 20)?,
+            }),
+            4 => Ok(FuzzOperation::AggregateVerifyMultiplePublicKeysMinPk {
+                public_keys: arbitrary_vec_g1(u, 0, 20)?,
+                namespace: arbitrary_optional_bytes(u, 50)?,
+                message: arbitrary_bytes(u, 0, 100)?,
+                signature: arbitrary_g2(u)?,
+            }),
+            5 => Ok(FuzzOperation::AggregateVerifyMultiplePublicKeysMinSig {
+                public_keys: arbitrary_vec_g2(u, 0, 20)?,
+                namespace: arbitrary_optional_bytes(u, 50)?,
+                message: arbitrary_bytes(u, 0, 100)?,
+                signature: arbitrary_g1(u)?,
+            }),
+            6 => Ok(FuzzOperation::AggregateVerifyMultipleMessagesMinPk {
+                public_key: arbitrary_g1(u)?,
+                messages: arbitrary_messages(u, 0, 20)?,
+                signature: arbitrary_g2(u)?,
+                concurrency: u.int_in_range(1..=8)?,
+            }),
+            7 => Ok(FuzzOperation::AggregateVerifyMultipleMessagesMinSig {
+                public_key: arbitrary_g2(u)?,
+                messages: arbitrary_messages(u, 0, 20)?,
+                signature: arbitrary_g1(u)?,
+                concurrency: u.int_in_range(1..=8)?,
+            }),
+            _ => Ok(FuzzOperation::PublicKeysMinPk {
+                public_keys: Vec::new(),
+            }),
+        }
+    }
+}
+
+fn arbitrary_g1(u: &mut Unstructured) -> Result<G1, arbitrary::Error> {
+    let bytes: [u8; 48] = u.arbitrary()?;
```

Comment:
nit: there are public consts for these numbers I think we should use:
* G1_ELEMENT_BYTE_LENGTH
* G2_ELEMENT_BYTE_LENGTH